lmdeploy/turbomind/supported_models.py

```diff
@@ -99,5 +101,7 @@ def _is_head_dim_128(cfg):
             elif arch == 'InternVLChatModel':
                 # internvl2-4b,internlm2-1b are not working yet
                 support_by_turbomind = _is_head_dim_128(cfg.llm_config)
+            elif arch == 'LlavaForConditionalGeneration':
+                support_by_turbomind = _is_head_dim_128(cfg.text_config)
```

Comment:
The conditions supported by turbomind should be that the top level model arch is `LlavaForConditionalGeneration` and the arch in `text_config` is `Qwen2ForCausalLM`